docs: Add debugging example (#202)
hostinger/fireactions: docs/examples/debugging-workflows.md

# Debugging workflows

> Sometimes you may need to connect to a running Fireactions VM in order to debug some workflow steps or look around in the host filesystem.

Reviewer: ```suggestion
Occasionally, it may be necessary to connect to a running Fireactions VM for debugging workflow steps or inspecting the filesystem.```

[tmate](https://github.com/mxschmitt/action-tmate) action provides a way to SSH into a running runner VM and have full access.

> Using it is as easy - just decide which workflow step to intercept, comment out the following steps and include tmate action above it:

Reviewer: ```suggestion
Using it is as easy - just decide which workflow step to intercept, comment out the subsequent steps, then insert the tmate action just before them:
```

```
    - name: Setup tmate session
      uses: mxschmitt/action-tmate@v3
```

> For security purposes, it's advised to add SSH keys to your github [profile](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) and limit access to yourself:

Reviewer: ```suggestion
For security purposes, it's advised to add SSH keys to your GitHub [profile](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) and limit access to yourself:
```

Also, this could be a note, use this formatting:
```
!!! warning
```
